nx.json

```diff
@@ -55,7 +48,33 @@
   "$schema": "./node_modules/nx/schemas/nx-schema.json",
   "targetDefaults": {
     "build": {
-      "dependsOn": ["^build"]
+      "dependsOn": ["^build"],
+      "inputs": ["production", "^production"]
+    },
+    "e2e": {
+      "inputs": ["default", "^production"]
+    },
+    "test": {
+      "inputs": ["default", "^production", "{workspaceRoot}/jest.preset.js"]
+    },
+    "lint": {
+      "inputs": ["default", "{workspaceRoot}/.eslintrc.json"]
+    },
+    "build-storybook": {
+      "inputs": ["default", "^production"]
     }
+  },
+  "namedInputs": {
+    "default": ["{projectRoot}/**/*", "sharedGlobals"],
+    "sharedGlobals": [],
```

Comment:
No :D 

All changes except the ones in `package.json` (and `yarn.lock`) are coming from Nx as automatic migrations. 

Digging in their changelog, and looking at the description in this PR https://github.com/nrwl/nx/pull/12416, it looks like `sharedGlobals` is used for "Various plugins that use babel". So my guess is that it's empty because we don't use Babel.